BigBaseV2/src/backend/looped/self/no_collision.cpp

```diff
@@ -0,0 +1,36 @@
+#include "backend/looped/looped.hpp"
+
+namespace big
+{
+	static bool bLastNoCollsion = false;
+
+	void looped::self_no_collision()
+	{
+		if (g_local_player == nullptr) return;
+
+		bool bNoCollsion = g->self.no_collision;
+
+		auto set_collison = [](float value) 
+		{
+			static int offsets[] = { 0x30, 0x10, 0x20, 0x70, 0x00, 0x2C };
```

Comment:
Let's not do that and use a struct instead

Reply:
Ok, will make a PR to GTAV Class in quick